pkg/lib/bundle/generate.go

```diff
@@ -43,8 +44,20 @@ type AnnotationMetadata struct {
 // @channels: The list of channels that bundle image belongs to
 // @channelDefault: The default channel for the bundle image
 // @overwrite: Boolean flag to enable overwriting annotations.yaml locally if existed
```

Comment:
nit: could you add a comment for the outputDir flag?

Reply:
+1